helpers/loanHelpers/userCollection.tsx

```diff
@@ -356,80 +190,135 @@ const calculateRisk = async (
   collection: string
 ) => {
   if (!obligations) return 0;
-  let filtered = await obligations.filter((obl: any) => obl.debt !== 0);
+  let filtered = await obligations.filter(
+    (obl: any) => obl.debt.toString() != 0
+  );
+
   let sumOfDebt = await filtered.reduce((acc: number, obligation: any) => {
-    return acc + obligation.debt;
+    return (acc += obligation.debt);
   }, 0);
 
   if (type === true) {
     return sumOfDebt;
   } else {
-    return sumOfDebt / filtered.length / nftPrice;
+    const sum = (sumOfDebt / filtered.length / nftPrice) * 100;
```

Comment:
was this *100 migrated from sdk to frontend?
verify if it wasnt correct at first/if its correct now

Reply:
it's the forumla for the risk factor, its the total sum of debt - div. by the length of the array div. by the nft price, which gives us a 00,XX% value - so we multiply by 100 to get a XX,XX% value